src/runtime/go-test.sh

```diff
@@ -0,0 +1,167 @@
+#!/bin/bash
+#
+# Copyright (c) 2017-2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set -e
+
+script_name=${0##*/}
+typeset -A long_options
+
+long_options=(
+	[help]="Show usage"
+	[package:]="Specify test package to run"
+)
+
+# Set up go test flags
+go_test_flags=${KATA_GO_TEST_FLAGS}
+if [ -z "$go_test_flags" ]; then
+    # KATA_GO_TEST_TIMEOUT can be set to any value accepted by
+    # "go test -timeout X"
+    go_test_flags="-timeout ${KATA_GO_TEST_TIMEOUT:-30s}"
```

Comment:
Another improvement suggestion: document in `usage()` the use of `KATA_GO_TEST_FLAGS` and `KATA_GO_TEST_TIMEOUT` variables.

Reply:
Not really in scope for me, sorry.